doc-content/shared-kie-docs/src/main/asciidoc/Install/proc-vscode-extension.adoc

```diff
@@ -1,10 +1,12 @@
 [id="proc-vscode-extension_{context}"]
 = Installing the {PRODUCT} VSCode extension bundle
 
-{PRODUCT} provides a *Red Hat Business Automation Bundle* VSCode extension that enables you to design Decision Model and Notation (DMN) decision models, Business Process Model and Notation (BPMN) 2.0 business processes, and test scenarios directly in VSCode. {PRODUCT} also provides individual *DMN Editor* and *BPMN Editor* VSCode extensions for DMN or BPMN support only, if needed.
+{PRODUCT} provides a *Red Hat Business Automation Bundle* VSCode extension that enables you to design Decision Model and Notation (DMN) decision models, Business Process Model and Notation (BPMN) 2.0 business processes, and test scenarios directly in VSCode. VSCode is the preferred integrated development environment (IDE) for developing new business applications. {PRODUCT} also provides individual *DMN Editor* and *BPMN Editor* VSCode extensions for DMN or BPMN support only, if needed.
+
+IMPORTANT: The editors in the VSCode are partially compatible with the editors in the {CENTRAL}, and several {CENTRAL} features are not supported in the VSCode.
 
 .Prerequisites
-* https://code.visualstudio.com/[VSCode] 1.46.0 or later is installed.
+* Latest stable version of https://code.visualstudio.com/[VSCode] is installed.
```

Comment:
I think we still go with complete sentences not telegraph style, so "The latest stable ..."

Reply:
Done